src/components/buttons/button-remove-image.jsx

```diff
@@ -0,0 +1,71 @@
+import ButtonCommand from '../base/button-command.js';
+import ButtonIcon from './button-icon.jsx';
+import EditorContext from '../../adapter/editor-context';
+import React from 'react';
+
+/**
+ * The ButtonRemoveImage class removes an image using a CKEDITOR.command.
+ *
+ * @class ButtonRemoveImage
+ * @uses ButtonCommand
+ */
+class ButtonRemoveImage extends React.Component {
+	static contextType = EditorContext;
+
+	static defaultProps = {
+		command: 'removeImage',
+		modifiesSelection: true,
+	};
+
+	static key = 'removeImage';
+
+	/**
+	 * @inheritDoc
+	 */
+	constructor(props) {
+		super(props);
+
+		const nativeEditor = props.editor.get('nativeEditor');
```

Comment:
To be consistent with the other buttons, we should be getting `editor` from the context and no the props. But the gotcha is that context is not available in a constructor without using our `EditorContext.toProps` hack, which obliges you to access the editor via `props.context.editor`.

Having said that, you don't see to need the editor here anyway, as it is getting passed in to the `exec` callback below.

Reply:
Ok, being consistent always sounds good, so I'll use our `EditorContext.toProps` method ... actually I need the `nativeEditor` to be able to call `addCommand` on it later.

I also noticed that this is usually done through plugins but I thought that for this, it wasn't really worth creating a plugin.

Reply:
@wincent I made some changes, please let me know what you think when you can (no rush)

 I can send another PR to avoid "polluting" the repo's history and logs, but I thought it would be nice to iterate on this until we think it's the right way to go.